sources/light_client.move

```diff
@@ -223,16 +243,25 @@ public(package) fun rollback(lc: &mut LightClient, checkpoint_hash: vector<u8>,
  * Views function
  */
 
-public fun latest_height(lc: &LightClient): u64 {
-    lc.finalized_height
+/// Returns height of the blockchain head (latest, not confirmed block).
+public fun head_height(lc: &LightClient): u64 {
+    lc.head_height
 }
 
+/// Returns height of the blockchain head (latest, not confirmed block).
+public fun head_hash(lc: &LightClient): vector<u8> {
+    lc.head_hash
+}
 
-public fun latest_block(lc: &LightClient): &LightBlock {
-    let block_hash = lc.get_block_hash_by_height(lc.latest_height());
-    lc.get_light_block_by_hash(block_hash)
+/// Returns blockchain head light block (latest, not confirmed block).
+public fun head(lc: &LightClient): &LightBlock {
+    lc.get_light_block_by_hash(lc.head_hash)
 }
 
+/// Returns latest finalized_block height
+public fun finalized_height(lc: &LightClient): u64 {
```

Comment:
Each application may have a different finality threshold. Can we make this "FINALITY" a function argument?